vorestation.dme

```diff
@@ -1647,9 +1647,14 @@
 #include "code\modules\ai\ai_holder_targeting_vr.dm"
 #include "code\modules\ai\interfaces.dm"
 #include "code\modules\ai\say_list.dm"
+<<<<<<< HEAD:vorestation.dme
 #include "code\modules\ai\ai_holder_subtypes\simple_mob_ai.dm"
 #include "code\modules\ai\ai_holder_subtypes\simple_mob_ai_vr.dm"
 #include "code\modules\ai\ai_holder_subtypes\slime_xenobio_ai.dm"
+=======
+#include "code\modules\ai\aI_holder_subtypes\simple_mob_ai.dm"
+#include "code\modules\ai\aI_holder_subtypes\slime_xenobio_ai.dm"
+>>>>>>> 21d0112... ice cream 2: electric boogaloo (#7319):polaris.dme

```

Comment:
..... Again with this naming convention issue, it seems.

Reply:
This is an issue with BYOND renaming it to aI_holder instead of ai_holder - we need our repo fixed to be aI_holder